test/src/main/java/org/apache/accumulo/test/functional/RecoveryWithEmptyRFileIT.java

```diff
@@ -95,7 +95,8 @@ public void replaceMissingRFile() throws Exception {
           Path rfile = StoredTabletFile.of(entry.getKey().getColumnQualifier()).getPath();
           log.debug("Removing rfile '{}'", rfile);
           cluster.getFileSystem().delete(rfile, false);
-          Process processInfo = cluster.exec(CreateEmpty.class, rfile.toString()).getProcess();
+          Process processInfo =
+              cluster.exec(CreateEmpty.class, "--type", "rfile", rfile.toString()).getProcess();
```

Comment:
For backwards compatibility, the rfile could be the default type if not specified.

Reply:
It is the default type and this worked without the change, but I thought that being specific here was better that relying on the default behavior.

Reply:
I was thinking this could be a nice coverage for the default case. As long as there's some coverage for the default case, I think either is fine here.

Reply:
Okay, I'll change it back to test the default.  At one point I was "why did this work?" and then realized that oh, yea, it is the default....  ;-)

Reply:
Reverted RecoveryWithEmptyRFileIT to use default in 2124f38165.  Also added a specific test in CreateEmptyTest to catch if the default changes without needing to run an IT.